WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -142,7 +142,7 @@ class AztecPostViewController: UIViewController, PostEditor {
     /// Separator View
     ///
     fileprivate(set) lazy var separatorView: UIView = {
-        let v = UIView(frame: CGRect(x: 0, y: 0, width: 44, height: 1))
+        let v = UIView(frame: CGRect(x: 0, y: 0, width: Constants.toolbarHeight, height: 1))
```

Comment:
This was already there, I was just applying the constant but I think this is to draw the bar to separate the plus (media) option from the other items?